apps/apps.c

```diff
@@ -207,6 +207,8 @@ static int ui_read(UI *ui, UI_STRING *uis)
         case UIT_INFO:
         case UIT_ERROR:
             break;
+        default:
+            break;
```

Comment:
It seems that default should not be necessary here (and other places), as all UIT constants are handled, and the return type from UI_get_string_type() is `enum UI_string_types`